lib/config/configuration.go

```diff
@@ -1448,11 +1452,26 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *service.Config) error {
 		}
 	}
 
+	if fc.WindowsDesktop.LDAP.PEMEncodedCACert != "" {
+		block, _ := pem.Decode([]byte(fc.WindowsDesktop.LDAP.PEMEncodedCACert))
```

Comment:
Couple of notes:
- We have `ParseCertificatePEM` method in lib/tlsca/parsegen.go you can use.
- Can the CA file contain multiple certificates? E.g. root-intermediate chain. If so, this would only load one which might result in X.509 errors. I have just fixed a similar issue for etcd backend: https://github.com/gravitational/teleport/pull/16484.

Reply:
Oh, I didn't know that `ParseCertificatePEM` exists! Thanks for showing me that function. 

As for the other point, to be honest with you I'm not sure if the CA certificate on the Active Directory server can contain multiple certificates. I'll double check that.